### ISSUE TITLE
Rebuild failure on atom 1.0.13

### ISSUE THREAD

**Issue by focusaurus** [pbihler/atom-macros#5]
Just updated to Atom 1.0.13 on OSX 10.10.5. After agreeing to the new XCode license to resolve the first build error, I see this one, which looks at a glance like atom-macros will need a fix.

```
In file included from ../src/main.cc:1:
In file included from ../src/common.h:6:
In file included from ../node_modules/nan/nan.h:74:
In file included from ../node_modules/nan/nan_new.h:190:
../node_modules/nan/nan_implementation_12_inl.h:181:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/plyons/.atom/.node-gyp/.node-gyp/0.30.6/deps/v8/include/v8.h:4515:3: note: 'New' declared here
  static Local<Signature> New(
  ^
1 error generated.
make: *** [Release/obj.target/pathwatcher/src/main.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "node" "/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/plyons/.atom/packages/atom-macros/node_modules/pathwatcher
gyp ERR! node -v v0.10.38
gyp ERR! node-gyp -v v2.0.2
gyp ERR! not ok 

npm ERR! Darwin 14.5.0
npm ERR! argv "/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/apm/bin/node" "/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/Users/plyons/.atom/.apm/.apmrc" "--userconfig" "/Users/plyons/.atom/.apmrc" "rebuild" "--target=0.30.6" "--arch=x64"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE
npm ERR! pathwatcher@4.4.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pathwatcher@4.4.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the pathwatcher package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls pathwatcher
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/plyons/.atom/packages/atom-macros/npm-debug.log
```

**Comment by focusaurus** [comment]
Sent pull request: https://github.com/pbihler/atom-macros/pull/6

**Comment by focusaurus** [comment]
Confirmed fixed in 0.5.2. Thanks!
